ansible/wireguard_sn3.yaml

```
@@ -521,3 +521,9 @@ wireguard_configs:
     PORT: 51978
     PEER_PUBLIC_KEY: zQv+ookRjHk1Gh6z468a7cZxfr+TDIyikh8yUPHEJBY=
     INTERFACE_ADDRESS: "10.70.250.114/31"
+
+    # primrose desktop
+  - NAME: primrosedesktop
+    PORT: 51979
+    PEER_PUBLIC_KEY: wbV5qUBN/w1dKRhbmcPiBL0R8B5GaSK74PQ2NzoSLTc=
+    INTERFACE_ADDRESS: "10.70.250.116/31"
```

I ran the workflow that checks the syntax, got this:

```
WARNING  Listing 1 violation(s) that are fatal
Error: No new line character at the end of file
```
Easy fix! 


```suggestion
    INTERFACE_ADDRESS: "10.70.250.116/31"
    
```